samples/core/get_started/eager.ipynb

```diff
@@ -8,6 +8,9 @@
       "views": {},
```

Comment:
Looks like a whole bunch of changes got pulled in... I will hold off reviewing until the changes are resolved, just in case the solution is, "delete this PR and start over."